ng2-qrcode/ng2-qrcode.ts

```diff
@@ -38,4 +53,16 @@ export class QRCodeComponent implements OnInit {
       console.error('Error generating QR Code: ' + e.message);
     }
   }
+
+  ngOnChanges(changes: {[propertyName: string]: SimpleChange}) {
```

Comment:
In the future we could support changing other options as well but after looking at the `QRCode` api it is not as straightforward. We'd need to destroy and recreate a child element that is bound to a `QRCode` instance I think.